### PR TITLE
[test_0]: HelloWorld pr for users-for #58.

### PR DESCRIPTION
**Pull request by Topwoo**
### What?

- brief: hello_word.cpp
- author: 1244807501@qq.com-Topwoo

### What changes are included in this PR?

包含了一个团队目录、个人目录、个人学习内容。

### Are these changes tested?

测试代码见main.cpp，里面编写了一个helloworld程序
image

### CODE REVIEW

File: stl/A_kuafu/Triplewo/src/00-hello_world/main.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include<iostream>
```
Comment (martinx) [inline stl/A_kuafu/Triplewo/src/00-hello_world/main.cpp:1]: 1 space is needed

```cpp
#include<iostream>
```

=> 

```cpp
#include <iostream>
```

File: stl/A_kuafu/Triplewo/src/00-hello_world/main.cpp [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+#include <string>
+using namespace std;
```
Comment (martinx) [inline stl/A_kuafu/Triplewo/src/00-hello_world/main.cpp:3]: You may not use a using-directive to make all names from a namespace available.

see: google style.https://google.github.io/styleguide/cppguide.html#Namespaces

**Review by martinx — COMMENTED** [review]
Please use lowercase for the directory

**Comment by Topwoo** [comment]
> 



> Please use lowercase for the directory

I have made changes according to the Google style. Could you please take a look？

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**